src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,55 @@
+import { FC } from "react";
+import styled, { css, useTheme } from "styled-components";
+
+import iconsMap from "./icons-map";
+
+export interface IconProps {
+  /**
+   * Id of the icon you want to use
+   */
+  name: keyof typeof iconsMap;
+  /**
+   * Which from the default sizes of the icon you want to use
+   */
+  size?: "small" | "medium" | "large";
+  /**
+   * Color of icon
+   */
+  color?: string;
+}
+
+const getIconSize = (size?: string) => {
+  switch (size) {
+    case "small":
+      return css`
+        height: 30px;
+      `;
+    case "large":
+      return css`
+        height: 80px;
+      `;
+    default:
+      return css`
+        height: 50px;
+      `;
+  }
+};
```

Comment:
Queste icon size sono dedotte dagli invision o sono arbitrarie? Personalmente lascerei la size custom facendo decidere a chi la usa la dimensione

Reply:
La 'small' l'ho presa da invision, le altre sono temporanee da editare in base alle necessità del mockup a utilizzo così da evitare di dover cambiarle ovunque nel caso di cambiamenti futuri. metto entrambi i parametri di input con height che sovrascrive la size, così abbiamo anche la versione custom per icon particolari.

Reply:
ottimo!